app/views/layouts/_google_analytics_script.haml

```diff
@@ -51,14 +51,15 @@
       ['b._trackPageLoadTime']
       );
 
-      (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-      })();
-
```

Comment:
If @current_community.google_analytics_key is nil, there's no analytics script added => we'll miss all the events.
(There's 2 ga tracking code one is for us and one is community specific -> both cases need to have this script put in place.)